### PR TITLE
fix(headless): insightFacet and insightDateFacet controllers set to use the correct analytics actions

### PR DESCRIPTION
**Pull request by mmitiche**
### The issue:
The `HeadlessInsightFacet` and `HeadlessInsightDateFacet` controllers are using the search analytics actions instead of the insight analytics actions.

The two controllers are using the analytics actions `logFacetDeselect` and `logFacetSelect` imported from `facet-set-analytics-actions` instead of the `facet-set-insight-analytics-actions` file.

### The solution:
The two controllers updated to use the analytics actions imported from the `facet-set-insight-analytics-actions` file.

### CODE REVIEW

**Review by btaillon — APPROVED** [review]

File: packages/headless/src/features/facets/range-facets/generic/range-facet-utils.ts [hunk 1]
```diff
@@ -20,3 +24,15 @@ export const getAnalyticsActionForToggleRangeFacetSelect = (
     ? logFacetDeselect(payload)
     : logFacetSelect(payload);
 };
+
+export const getInsightAnalyticsActionForToggleRangeFacetSelect = (
```
Comment (ThibodeauJF) [inline packages/headless/src/features/facets/range-facets/generic/range-facet-utils.ts:28]: Would be best if it's in its own file correctly scoped to **insight** so things don't get too convoluted

File: packages/headless/src/features/facets/range-facets/generic/range-facet-utils.ts [hunk 2]
```diff
@@ -3,6 +3,10 @@ import {
   logFacetDeselect,
   logFacetSelect,
 } from '../../facet-set/facet-set-analytics-actions';
+import {
+  logFacetDeselect as logInsightFacetDeselect,
+  logFacetSelect as logInsightFacetSelect,
+} from '../../facet-set/facet-set-analytics-actions';
```
Comment (ThibodeauJF) [inline packages/headless/src/features/facets/range-facets/generic/range-facet-utils.ts:9]: Description says 

> The two controllers updated to use the analytics actions imported from the facet-set-insight-analytics-actions file.

But here it's importing the same facet-set-analytics-actions file

Comment (mmitiche) [reply packages/headless/src/features/facets/range-facets/generic/range-facet-utils.ts]: My bad 😅I just updated the import, thank you!

**Review by aelfaleh — APPROVED** [review]
LGTM

**Review by nathanlb — APPROVED** [review]
gg

**Pull request merged**